### ISSUE TITLE
loaded model ground truth labels

### ISSUE THREAD

**Issue by jalilahmed** [keras-team/keras#9985]
I have loaded a model using, 
model = load_model('weights.h5')
prediction = model.predict(image)

the prediction returns me a numpy array of class probabilities, but I dont know which probability belongs to which class. how to figure it out??

**Comment by Dref360** [comment]
It's the same order as when you created the model.

This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.

To ask questions, please see the following resources : 
* https://keras.io/
* https://keras.io/#support
* https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please re-open this issue.
